Core/Parsing/ExpressionVisitors/TreeEvaluation/EvaluatableTreeFindingExpressionVisitor.cs

```diff
@@ -61,6 +61,9 @@ public sealed class EvaluatableTreeFindingExpressionVisitor : RelinqExpressionVi
       return visitor._partialEvaluationInfo;
     }
 
+    // PERF: Avoids using Enum.IsDefined, which is really slow.
+    private static readonly ExpressionType s_maxExpressionType = Enum.GetValues(typeof (ExpressionType)).Cast<ExpressionType>().Max();
```

Comment:
Please add a whitespace after Enum.GetValues. We always keep a space when calling methods with parameters.